### PR TITLE
Chore(wren-ui): update wren-ui readme

### PR DESCRIPTION
**Pull request by onlyjackfrost**
Provide local setup instruction and FAQ

### CODE REVIEW

File: wren-ui/README.md [hunk 1]
```diff
@@ -55,6 +69,66 @@ You can start editing the page by modifying `app/page.tsx`. The page auto-update
...
+
+Step 4. Start wren-ui from source code
+refer to [Start wren-ui from source code](#Start-wren-ui-from-source-code) section to start wren-ui from source code.
+
+## FAQ
+### Can I have multiple project at the same time in Wren AI?
+We currently do not support multiple projects in Wren AI. You can only have one project at a time.
+But there is a workaround for this. Since Wren Engine is stateless and we store your semantic model in the database(Sqlite or Postgres), 
+you can switch between projects by switching the database and make sure you deploying after server started.
+
```
Comment (fredalai) [inline wren-ui/README.md:107]: (NIT) Suggestion:
Add a tip to describe a brief description - the key point is to use the **SQLITE_FILE** variable to change the DB

```markdown
> Tip: Define the `SQLITE_FILE` variable to specify the path to the database you intend to use.
```

File: wren-ui/README.md [hunk 2]
```diff
@@ -55,6 +69,73 @@ You can start editing the page by modifying `app/page.tsx`. The page auto-update
...
+docker-compose -f docker-compose-dev.yaml --env-file .env.example up -d
+# then stop the services via
+docker-compose -f docker-compose-dev.yaml --env-file .env.example down
+```
+
+Step 4. Start wren-ui from source code
+refer to [Start wren-ui from source code](#Start-wren-ui-from-source-code) section to start wren-ui from source code.
+
+Step 5. (Optional) Develop other modules along with wren-ui
+As mentioned above, you can use docker-compose to start other modules. The same applies when developing other modules.
```
Comment (fredalai) [inline wren-ui/README.md:103]: Add an extra blank line if you want it to wrap. 
So are other places.

File: wren-ui/README.md [hunk 3]
```diff
@@ -55,6 +69,73 @@ You can start editing the page by modifying `app/page.tsx`. The page auto-update
...
+From the perspective of wren-ui, if you want to develop other modules at the same time, you can stop the container then spin up the module from the source code.
+Refer to the README.md or CONTRIBUTION.md file the module for starting the module from the source code.
```
Comment (fredalai) [inline wren-ui/README.md:105]: For README.md and CONTRIBUTION.md, do we have a link to provide?

File: wren-ui/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-## Getting Started
+## Start wren-ui from source code
```
Comment (fredalai) [inline wren-ui/README.md:3]: (NIT) Suggestion:
Also can use **Local development** or **Developing**

**Review by inductor69 — APPROVED** [review]

**Review by wwwy3y3 — APPROVED** [review]

**Pull request merged**